### DISCUSSION TITLE
Bem-Vindo as discussões da BibliotecaDev!

### DISCUSSION THREAD

**Discussion by KAYOKG** [KAYOKG/BibliotecaDev#3] [Announcements]
### Usem as discussões para:

-    Transmitir seu entendimento sobre algum livro;
-    Sugerir algum livro para acrescentar na biblioteca e sugestão em qual categoria ele se encaixa;
-    Discutir sobre os livros;

 Para começar, comente abaixo sobre os livros que você tem interesse e os que você irá começar ou ja começou a ler.

**Comment by orlandojsjunior** [comment]
Olá boa tarde, tenho alguns livros em pdf e gostaria de contribuir com o repositório.

**Reply by KAYOKG** [reply]
Olá boa tarde, seria quais livros você tem a agregar? a biblioteca ela tem como foco bons livros e que não sejam livros maçantes, de conteúdo pesado, mas compartilhe conosco quais livros seriam esses.

**Reply by orlandojsjunior** [reply]
01 - Python 3 conceitos e aplicações - uma abordagem didática
02 - Python para análise de dados - by Wes McKinney 
03 - Introdução à programação com Python – 3ª edição -- Nilo Ney Coutinho Menezes 2019
04 - Estatística - O que e, para que serve, como funciona 
05 - Estatística prática para cientistas de dados_ 50 conceitos -- Andrew Bruce & Peter Bruce - 2019 - Alta Books

**Reply by KAYOKG** [reply]
Legal, adicione um Pull Request ao projeto. porém a biblioteca não tem o foco em linguagens de programação, por exemplo adicionar muitos livros de python, mas no caso de análise de dados seria ideal criar uma nova categoria para isso, como Data Science.

**Comment by DeiseFreire** [comment]
Thank you for sharing your book library with us. That is very kind!

**Comment by orlandojsjunior** [comment]
Boa noite, estava meio ocupado com as aulas da faculdade e dos cursos, mas amanhã irei add os livros de Data Science na biblioteca.
